sellercenter-category/src/main/java/com/gfg/sellercenter/category/entity/CategoryDetails.java

```diff
@@ -0,0 +1,32 @@
+package com.gfg.sellercenter.category.entity;
+
+import lombok.*;
+
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+public class CategoryDetails implements Serializable {
+    private int id;
```

Comment:
a little bit of space would be nice for all attributes.

Reply:
Did like you suggested, however I was not able to find that it is required in our style guide https://google.github.io/styleguide/javaguide.html . Is it?

Reply:
No clue really but now it is more readable at least.

Reply:
Thanks. Once there is nothing about it in styleguide, in future I will also use blank lines (I guess we were doing like that in PHP and it is required)

Reply:
yes